flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
disappointing

Reply:
Let's give it a try. I think https://build.opensuse.org/package/live_build_log/games/warsow/openSUSE_Tumbleweed/aarch64 is failing for other reasons.

Reply:
```
../../source/as_callfunc_x64_gcc.cpp: In function ‘asQWORD X64_CallFunction(const asQWORD*, int, funcptr_t, asQWORD&, bool)’:
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%rcx’ in ‘asm’
   74 |         __asm__ __volatile__ (
      |         ^~~~~~~
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%rdx’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%rax’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%rsi’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%rdi’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm7’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm6’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm5’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm4’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm3’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm2’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm1’ in ‘asm’
../../source/as_callfunc_x64_gcc.cpp:74:9: error: unknown register name ‘%xmm0’ in ‘asm’
make[3]: *** [makefile:125: obj/as_callfunc_x64_gcc.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/angelscript.dir/build.make:70: CMakeFiles/angelscript] Error 2
make[1]: *** [CMakeFiles/Makefile2:334: CMakeFiles/angelscript.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

The builder also fails to abort and get stuck on arm:

> flatpak-builder: Cleanup warsowError: module warsow: Child process exited with code 2

Reply:
Fedora builds it from source and has some patches available: https://src.fedoraproject.org/rpms/warsow/tree/rawhide

I tend to be practical about binary vs source builds, but I don't see the advantages for the binary in this case.

Reply:
I also build it from source using the same tarball.